### DISCUSSION TITLE
What else can I contribute to reducing issues besides PR?

### DISCUSSION THREAD

**Discussion by yamatatsu** [aws/aws-cdk#17426] [General]
For example, report an Issue that appears to be resolved to someone else.

I've already found some issues that can be closed.

- https://github.com/aws/aws-cdk/issues/9927
- https://github.com/aws/aws-cdk/issues/12198
- https://github.com/aws/aws-cdk/issues/5420
- https://github.com/aws/aws-cdk/issues/16339

Who should I report this to in a similar case? Or could these reports be noisy for maintainers?

**Comment by skinny85** [comment]
Generally, commenting on them saying "this was done in PR `<link to PR>`"  is good enough 🙂.
